### ISSUE TITLE
Get quote for 20 miRNA libraries

### ISSUE THREAD

**Issue by sr320** [RobertsLab/resources#1902]
Manilla clam  is species

**Comment by kubu4** [comment]
How much sequencing depth should we aim for? 20x? 10x?

**Comment by sr320** [comment]
20x

**Comment by kubu4** [comment]
Azenta and Genohub quote requests submitted.

**Comment by kubu4** [comment]
Waiting on response to see if they can do sRNA-seq library construction.

**Comment by kubu4** [comment]
GenoHub quote: `$4,100.00`

A few things to note:

- I deferred to them on recommended sequencing parameters for sRNA-seq.
- 1 x 50bp reads
- 10M average reads/sample
- Using a Singular Genomics G4 sequencer (!?!)

[Genohub_Quote_54151821_Confirmed.pdf](https://github.com/RobertsLab/resources/files/15394232/Genohub_Quote_54151821_Confirmed.pdf)

**Comment by kubu4** [comment]
UW has confirmed that they do not make sRNA-seq libraries.

**Comment by kubu4** [comment]
Azenta: `$5,610`

A few things to note:

- Using Azenta recommendations for sequencing.
- 2 x 150bp reads
- 20M average reads/sample


[30-1035633055_R1.pdf](https://github.com/RobertsLab/resources/files/15436203/30-1035633055_R1.pdf)

**Comment by sr320** [comment]
Let's go with azenta - @mgavery can you select samples?

**Comment by kubu4** [comment]
@mgavery - Change of plans for me. I won't be able to make it into tomorrow. Sorry about this!

Not sure if I'll be able to make it in on Thursday, either, but will update when I know more.

**Comment by kubu4** [comment]
Just following up to see the status on this. I'm guessing no one else coordinated with @mgavery to get samples?

If not, I'm planning on being in tomorrow and Thursday of this week. Please let me know if either of those work.

**Comment by mgavery** [comment]
Hi, Tuesday works for me.  Any time after 10am.

**Comment by mgavery** [comment]
samples provided: 
196,199,211,310,341,363,367,376,460,485
24,26,30,33,34,35,71,88,260,501

**Comment by kubu4** [comment]
@sr320 - Turns out these are tissue samples.

The quote I got was for total RNA.

Should I:

- Get updated quote, with tissue as starting material?

OR

- Plan on isolating RNA?

**Comment by kubu4** [comment]
Per discussion at Hack-a-thon today, will get updated quote for tissue as starting material.

**Comment by kubu4** [comment]
@mgavery - Which tissue are these?

**Comment by kubu4** [comment]
@mgavery - NM! I see it on the box (eggs).

**Comment by kubu4** [comment]
Updated Azenta quote:

`$6,310` (~$315/sample)

[30-1035633055_R2.pdf](https://github.com/user-attachments/files/15635336/30-1035633055_R2.pdf)


@sr320 - Proceed with updated quote?

**Comment by sr320** [comment]
> Proceed with updated quote?

yes

**Comment by sr320** [comment]
I added budget info in slack.

**Comment by kubu4** [comment]
FYI - Looks like PO request was awaiting Lisa Smith's approval and was approved yesterday (7/14/2024). Now awaiting approval from cost center manager. For the record, PO request was initially submitted on 6/26/2024.

**Comment by mgavery** [comment]
[30-1035633055_NGS_Template_MG.xls](https://github.com/user-attachments/files/16314665/30-1035633055_NGS_Template_MG.xls)

Honestly, I don't remember how these are labeled on the top of the tubes.  I think the ID# is preceded by "M-", but I'm not positive. Could you do a quick check and update if I messed it up?

**Comment by kubu4** [comment]
> Could you do a quick check and update if I messed it up?

Will do!

**Comment by kubu4** [comment]
> samples provided: 196,199,211,310,341,363,367,376,460,485 24,26,30,33,34,35,71,88,260,501

@mgavery - These do not match the sample names you entered in the [sample sheet](https://github.com/RobertsLab/resources/issues/1902#issuecomment-2239615948)??

**Comment by kubu4** [comment]
Chatted with @mgavery via Slack.

She's indicated to use update sample sheet to use tube numbers.
